charms/istio-pilot/actions.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+set-tls-manually:
+  description: |
+    Manually pass SSL cert and key values to configure the Ingress Gateway with TLS.
+    Configuring TLS with this action is mutually exclusive to using TLS certificate providers.
+  params:
+    ssl-key:
+      type: string
+      description: |
+        The SSL key output as a string. Can be set with
+        $ juju run set-tls-manually istio-pilot/<unit-number> ssl-key="$(cat KEY_FILE)"
+    ssl-crt:
+      type: string
+      description: |
+        The SSL cert output as a string. Can be set with
+        $ juju run set-tls-manually istio-pilot/<unit-number> ssl-crt="$(cat CERT_FILE)"
```

Comment:
same as above: let's move the example to the description of the action and change it to set both the key and cert since now both inputs are required